cve_bin_tool/cli.py

```diff
@@ -293,11 +298,11 @@ def main(argv=None):
         prog="cve-bin-tool",
         description=textwrap.dedent(
             """
-            The CVE Binary Tool scans for a number of common, vulnerable open source
-            components (openssl, libpng, libxml2, expat and a few others) to let you know
-            if a given directory or binary file includes common libraries with known
-            vulnerabilities.
-            """
+                    The CVE Binary Tool scans for a number of common, vulnerable open source
```

Comment:
What happened here?  This is going to change the --help output and I don't think it's going to look right.

Reply:
If this is due to your development environment's auto-formatting, you may need to turn it off.

Reply:
Yeah don't merge it. First it should be in scanner because in scanner we are actually loading checker and second I think i can suppress that output. 
I will do it tomorrow because i am going to bed 😴 now.